sys/arm/broadcom/bcm2835/bcm2835_gpio.c

```diff
@@ -151,8 +159,12 @@ enum bcm_gpio_pud {
 #define	BCM_GPIO_GPLEN(_bank)	(0x70 + _bank * 4)	/* Low Level irq */
 #define	BCM_GPIO_GPAREN(_bank)	(0x7c + _bank * 4)	/* Async Rising Edge */
 #define	BCM_GPIO_GPAFEN(_bank)	(0x88 + _bank * 4)	/* Async Falling Egde */
-#define	BCM_GPIO_GPPUD(_bank)	(0x94)			/* Pin Pull up/down */
-#define	BCM_GPIO_GPPUDCLK(_bank) (0x98 + _bank * 4)	/* Pin Pull up clock */
+#define	BCM2835_GPIO_GPPUD(_bank) (0x94)		/* Pin Pull up/down */
+#define	BCM2835_GPIO_GPPUDCLK(_bank) (0x98 + _bank * 4)	/* Pin Pull up clock */
+
+#define	BCM2711_GPIO_GPPUD(x)	(0x0e4 + (x) * sizeof(uint32_t)) /* Pin Pull up/down */
+#define	BCM2711_GPIO_MASK(n)	(0x3 << ((n) % 16)*2)
```

Comment:
If you add:
```
#define BCM2711_GPIO_PIN_SHIFT(n) (((n) % 16) * 2)
```

Then you can use it as part of this definition, and below (see my other comment).